.github/workflows/push.yml

```diff
@@ -135,7 +135,7 @@ jobs:
           make checks
           git diff --exit-code
 
-  validate-bundle-schema:
+  validate-generated-is-up-to-date:
```

Comment:
If this is changed then we also need to change repository settings since this is one of the required checks.